src-frontend/app/components/page-list.js

```diff
@@ -10,13 +10,19 @@ export default Ember.Component.extend({
             scroll: true,
             placeholder: 'ui-state-highlight',
             update: function(event, ui) {
-                var indices = {};
+                var sortOrder = $(this).sortable('toArray');
```

Comment:
Forgot to mention this, but can we use `Ember.$` instead of `$`? This means you can remove the `import $` from the top